src/components/Pages/Home/LoggedOutFeatures/GetStartedBanner.vue

```diff
@@ -3,7 +3,7 @@
     <v-row>
       <v-col cols="12" md="6">
         <v-row>
-          <h1 class="header-main">Welcome to wikibase.cloud</h1>
```

Comment:
This change isn't used and confused me a bit, I think we can just remove this file entirely.